mantidproject/mantid
# Run *.png through zopfli lossless compression

**Description of work.**
Runs all png files through zopfli lossless compression with the most
intense settings:
find -iname *.png -print0 | xargs -0 -n 1 -P12 -I{} zopflipng -y --iterations=500 --filters=01234mepb --lossy_transparent {} {}

This reduced the size of the repo from 106MB for 63MB as determined by:
find . -type f -iname '*.png' -exec du -ch {} + | grep total$

This will directly reduce the package size by this amount too, as LZMA
and similar compression methods don't attempt to re-compress png usually

**To test:**
- Build docs locally / open Mantid
- Check that a random selection of icons / screenshots and images open correctly

There is no associated issue - Completed as I already had the script to hand for a personal project

*This does not require release notes* because the compression is lossless and therefore transparent



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
